.github/workflows/build.yml

```diff
@@ -1,10 +1,6 @@
 name: Build
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+on: [push, pull_request]
```

Comment:
Not sure of the exact semantics, but doesn't push imply pull request?

Reply:
Oh, yes, you are right that having the `pull_request` still there without the restriction to one branch (which was the main purpose of this specific change, I wanted to test the explicit compiler specification on a branch of my repository) does probably not make much of a difference any more. Usually anything for which a pull request is opened should have been checked on the push to the source branch before, although as far as I can see, it might cause checks that would otherwise not happen on events like a pull request from a repository with a different configuration or a reopened pull request without any new changes, but I can't imagine that ever being relevant for us.